.circleci/config.yml

```diff
@@ -359,6 +368,10 @@ jobs:
           name: Create purchases integration tests apks
           command: |
             bundle exec fastlane android build_load_shedder_purchases_integration_tests
+      - run:
+          command: |
+            bundle exec fastlane android send_slack_load_shedder_integration_test success:false
+          when: on_fail
```

Comment:
There is no way to specify the on_fail/on_success at the workflow level in CircleCI. You need to set this up at the job level.

Reply:
We could do it for that job easily (building the APKs) since that's executed in fastlane. But executing the integration tests in firebase test labs is not currently run through fastlane so it's much harder.

We could potentially move the command that executes the firebase tests to fastlane, but that assumes that the gcloud cli is installed and configured, and this currently happens on different executors... 

I will try to see if I can install the gcloud cli and configure it in the android executor instead... That way, we could execute everything as a single job and move all this logic to fastlane. It's a bigger change, but it's good to move more logic to fastlane